### PR TITLE
feat : Added Toastify

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added Toast Notify

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Easy review due to small PR 
Clean code and UI 
Good for approval

File: src/store/userSlice.js [hunk 1]
```diff
@@ -12,6 +13,7 @@ export const followUser = createAsyncThunk(
   async ({ followUserId, authToken, dispatch }) => {
     const response = await followUserServices(followUserId, authToken);
     dispatch(editProfile({ userDetails: response.data.user, token: authToken }));
+    ToastHandler('success', `You are noe following ${response.data.followUser.username}`);
```
Comment (Nishant-96) [inline src/store/userSlice.js]: Spelling mistake.

**Review by Nishant-96 — APPROVED** [review]
Functionality working as expected.
- Liked the UI.
- Easily understandable code.
- Found a Bug: Able to create empty posts.

File: src/store/userSlice.js (hunk 1 above)
Comment (anuj-kumary) [reply src/store/userSlice.js]: Done

**Pull request merged**